llvm/lib/ProfileData/MemProfReader.cpp

```diff
@@ -40,6 +40,71 @@
 #include "llvm/Support/Path.h"
 
 #define DEBUG_TYPE "memprof"
+
+namespace llvm {
+namespace yaml {
+template <> struct MappingTraits<memprof::Frame> {
+  static void mapping(IO &Io, memprof::Frame &F) {
```

Comment:
It would be nice to ensure this remains consistent with the definition of Frame. Maybe a static_assert of some sort might help?

Reply:
Fixed in the latest iteration.